index.rst

```diff
@@ -1,11 +1,17 @@
-This is the main page for the PLEXIL wiki. For PLEXIL's main project
-page, which has downloads and other information, go here:
-https://github.com/plexil-group/plexil
+This is the main page of the PLEXIL User's Guide. It documents PLEXIL
+Release 4.6, which is the version of PLEXIL found on its `main` Git

```

Comment:
The 'main' branch only exists on Babelfish. On GitHub, it's releases/plexil-4.6

Reply:
Fixed.